doc/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:alpine
+WORKDIR /data/
+# Fix proxy issues
+RUN apk update && apk --no-cache add openssl wget
+RUN wget https://github.com/vuejs/vuepress/archive/master.tar.gz
```

Comment:
why are you getting vuepress and extracting it.

Reply:
If you are already using yarn, just download it using yarn and it will lock you into a version preventing possibilities of version clashes/build fails in a CD environment 👍

Reply:
Now uses yarn/vuepress@next.